test/Exchange.Matching.test.js

```diff
@@ -12,13 +12,19 @@ if (config.LOG) {
 
 const PPM_DIV = 1000000;
 
-function getBnPrice(price) {
-    return new BigNumber((price * PPM_DIV).toFixed(0));
+function toPPM(price) {
+    return new BN(price * PPM_DIV);
+}
+
+
+const ONE_ETH_IN_WEI = new BN("1000000000000000000");
+
+function toWei(amount) {
```

Comment:
can we move this (and `toPPM` / `fromPPM` as well) to a static `Augmint.utils.toWei` ? or `utils.conversions` (or similar just shorter name for the lazy). And without the `PPM_DIV` hack :)

Reply:
Apparently I'm unable to do, but I fully support the idea :) Let's do this in a separate PR and merge this one.

Reply:
let's wait for @rszaloki  :)